retrofit-converters/gson/src/test/java/retrofit2/converter/gson/GsonConverterFactoryTest.java

```diff
@@ -78,7 +78,7 @@
   }
 
   interface Service {
-    @POST("/") Call<AnImplementation> anImplementation(@Body AnImplementation impl);
+    @POST("/") Call<AnImplementation> anImplementation(@Body(ignoreNull = false) AnImplementation impl);
```

Comment:
this is kind of hard to read ’cause of the double negative. Should it be `nullable = true` ?